private/pkg/protosource/field.go

```diff
@@ -33,6 +33,7 @@ type field struct {
 	jsType         FieldOptionsJSType
 	cType          FieldOptionsCType
 	packed         *bool
+	deprecated     *bool
```

Comment:
Why is this one a pointer, but the others are not?

Note that it is possible to set `[deprecated=false]` in protobuf, and using a pointer would preserve this case. I don't think we should do that here, though. Descriptors are not an accurate representation of the protobuf source, and deprecated = false is not meaningful.

Reply:
I agree that the `deprecated = false` is not meaningful for us here, hence no reason to use a pointer. I'll change it.